packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```diff
@@ -0,0 +1,16 @@
+@import '../../../global/global.pcss';
+@import '../facet-common.pcss';
+@import '../facet-search/facet-search.pcss';
+
+.box-container {
+  @apply flex items-stretch inline-block;
```

Comment:
If you apply 
`display: flex` then it will be overridden by the `display: inline-block`
https://tailwindcss.com/docs/display

Reply:
For some reason the opposite happens. Flex and items-stretch is applied, but not inline-block

Reply:
could it be because this class hasn't been created yet and you need to restart the server? In a PR that I will (hopefully) merge today I added `mode: 'jit'` to tailwind config so you don't have to restart the server every time you use a class that hasnt been used before

Reply:
oh really? that's weird, I have that issue all the time...

Reply:
does restarting the server make that class work though @mbenkirane-coveo ?

Reply:
No it doesn't change anything